src/components/Dropdown/Dropdown.tsx

```diff
@@ -141,12 +133,14 @@ export const Dropdown: FC<DropdownProps> = ({
 
     const showClear = !!activeItem && !!onClear;
 
-    const getDropdownBottomPosition = (dropDownSize: string) => {
-        return dropDownSize === DropdownSize.Small ? "tw-mb-2 tw-bottom-[34px]" : "tw-mb-2 tw-bottom-[60px]";
-    };
+    const leftPosition =
+        triggerRef.current && overlayRef.current
+            ? triggerRef.current?.getBoundingClientRect().right - overlayRef.current?.getBoundingClientRect().width
+            : "auto";
+    const bottomPosition = triggerRef.current ? triggerRef.current?.getBoundingClientRect().top - 8 : "auto";
```

Comment:
Spacing between trigger and flyout is different for top and bottom position.
Please make sure to align with [the figma design](https://www.figma.com/file/pevuvey1joCsopTRr9f4OE/Fondue-React-Components?node-id=13839%3A254266)
![Screenshot 2022-04-11 at 10 37 26](https://user-images.githubusercontent.com/3276065/162697959-618cc3d0-4fba-4e20-bf03-5ca0a69c3ca0.png)
![Screenshot 2022-04-11 at 10 37 19](https://user-images.githubusercontent.com/3276065/162697963-344cf54c-ccac-4cd8-bc13-1039d2f61fd2.png)